pkg/cache/store.go

```diff
@@ -37,6 +38,11 @@ func (cs *Store) DelBlob(blob string) error {
 		return errors.Wrapf(err, "remove blob %v err", blobPath)
```

Comment:
Even blob and chunk map deletion fails, we still try to remove blob.meta?

Reply:
It would be safer to keep the blob.meta file in such a case.